src/world/ObjectMgr.cpp

```diff
@@ -3898,4 +3898,172 @@ void ObjectMgr::LoadAreaTrigger()
     while (result->NextRow());
 
     Log.Success("AreaTrigger", "Loaded %u area trigger teleport definitions", count);
+}
+
+void ObjectMgr::LoadEventScripts()
+{
+    Log.Notice("ObjectMgr", "Loading Event Scripts...");
+
+    QueryResult* result = WorldDatabase.Query("SELECT * FROM event_scripts WHERE event_id > 0 ORDER BY event_id");
```

Comment:
Using a wildcard select is dangerous as it can cause unrelated future database changes to break your query in a potentially non-obvious way. In queries, ensure that each column you'd like to select is specifically named.

```
const char* eventScriptsQuery = "SELECT event_id, function, script_type, data_1, data_2, data_3, data_4, data_5, x, y, z, o, delay, next_event FROM event_scripts WHERE event_id > 0 ORDER BY event_id";
```